content.org

```diff
@@ -284,6 +297,26 @@ Evaluate some normal functions in your REPL
   15
   #+END_SRC
 
+*** Structural Editing
+
+Structural editing is a method of manipulating code based on its syntactic structure, 
+rather than its text representation.
+It allows for operations like moving, wrapping or editing code blocks as discrete units.
+
+Plugins that enable structural editing:
+
+ - Visual Studio Code: Calva (https://calva.io/)
+ - Intellij: Cursive (https://cursive-ide.com/)
+
+Common Paredits for Visual Studio: 
+
+| Action                     | Windows                | Linux      | Mac                     |
+| -------------------------- | ---------------------- | ---------- | ------------------------|
+| Slurp Forward              | ctrl+alt+right         | ctrl+alt+. | ctrl+opt+right          |
+| Barf Forward               | ctrl+alt+left          | ctrl+alt+, | ctrl+opt+left           |
+| Raise Sexp                 |       ctrl+alt+p  ctrl+alt+r        | ctrl+opt+p  ctrl+opt+r  |
+| Drag sexp forward/backward |           alt+up/down arrows        | opt+up/down arrows      |
```

Comment:
try fixing the table here by writing something like
| Raise Sexp                 |       ctrl+alt+p  ctrl+alt+r       || ctrl+opt+p  ctrl+opt+r  |